### PR TITLE
rtloader: drop python2 support

### PR DESCRIPTION
**Pull request by chouquette**
### What does this PR do?

drop python2 support from rtloader

### Motivation

Removing python2 support from our codebase since we don't support it anymore

### CODE REVIEW

**Review by kacper-murzyn — APPROVED** [review]

File: rtloader/README.md [hunk 1]
```diff
@@ -15,14 +15,13 @@ RtLoader will `dlopen` the proper backend libraries accordingly.
...
-using the `make2` and `make3` functions, the corresponding Python backend will
+using the `make3` function, the corresponding Python backend will
 be loaded at runtime. Under the hood the library provides `RtLoader`, a C++ interface
 that must be implemented by any supported backend, see `include/rtloader.h` for details.
 
 ### Two and Three
 
-`libdatadog-agent-three` and `libdatadog-agent-two` libraries provide Python support
-for extending and embedding by linking different versions of the CPython library.
+`libdatadog-agent-two` library provides Python3 support. Python2 isn't supported anymore.
```
Comment (vickenty) [inline rtloader/README.md:24]:
```suggestion
`libdatadog-agent-tree` library provides Python3 support. Python2 isn't supported anymore.
```

File: rtloader/include/datadog_agent_rtloader.h [hunk 2]
```diff
@@ -28,17 +28,6 @@ typedef struct rtloader_s rtloader_t;
 struct rtloader_pyobject_s;
 typedef struct rtloader_pyobject_s rtloader_pyobject_t;
 
-// FACTORIES
```
Comment (vickenty) [inline rtloader/include/datadog_agent_rtloader.h:31]: Would it make sense to keep this line? I think it applies to both `make2` and `make3`.

**Review by vickenty — APPROVED** [review]

File: rtloader/README.md (hunk 1 above)
Comment (chouquette) [reply rtloader/README.md:24]: woops, good catch, thanks!

**Pull request merged**